test/utils/form.ts

```diff
@@ -0,0 +1,296 @@
+import { Locator } from '@playwright/test';
+import { DashboardPage, expect, Panel, PanelEditPage } from '@grafana/plugin-e2e';
+import { getLocatorSelectors, LocatorSelectors } from './selectors';
+import { TEST_IDS } from '../../src/constants/tests';
+
+const getElementsSelector = getLocatorSelectors(TEST_IDS.formElements);
+
+/**
+ * Disabled Element Helper
+ */
+class DisabledElementHelper {
+  private readonly locator: Locator;
+
+  constructor(parentLocator: Locator) {
+    this.locator = parentLocator.getByTestId(TEST_IDS.formElements.fieldDisabled);
+  }
+
+  private getMsg(message: string): string {
+    return `DisabledElement: ${message}`;
+  }
+
+  public get() {
+    return this.locator;
+  }
+
+  public async checkValue(text: string) {
+    return expect(this.get(), this.getMsg('Check Text')).toHaveValue(text);
+  }
+}
+
+/**
+ * Number Element Helper
+ */
+class NumberElementHelper {
```

Comment:
I would create a base element helper which implements all basic scenarios like checkValue, setValue, etc.
If element has some specific logic, then it can be overwitten in its helper

```
class BaseElementHelper {
  public setValue () {}
  public checkValue () {}
}

class DisabledElementHelper extends BaseElementHelper {}

class NumberElementHelper extends BaseElementHelper {
  public setValue(value) {
    this.element.fill(Number(value))
  }
}

```